<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logPage</class>
 <widget class="QMainWindow" name="logPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>428</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CookBook</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="h1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добро пожаловать</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="log_in_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="log_out_btn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Я - новичок</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>логин</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passw_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>пароль</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorHandler">
     <property name="geometry">
      <rect>
       <x>36</x>
       <y>80</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(237, 51, 59);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="h2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>все рецепты мира в одном девайсе</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>251</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(221, 221, 221);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="h1_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="h2_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>** заполните все поля **</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="back_to_login_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QPushButton" name="log_out_btn_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_logout_le">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>логин</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passw_logout_le">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>пароль</string>
      </property>
     </widget>
     <widget class="QLabel" name="errorHandler_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(237, 51, 59);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_logout_le">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>имя</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
